### ISSUE TITLE
Use Multiboot Header to Get RAM Size

### ISSUE THREAD

**Issue by Kfeavel** [panix-os-dev-team/Panix#140]
We should at least grab the amount of available memory out of the multiboot header even if we don't use it right now. It is something we should consider in the future.

```C
size_t total_memory = 0;
mmap_entry_t* entry = mbt->mmap_addr;
while(entry < mbt->mmap_addr + mbt->mmap_length) {
	entry = (mmap_entry_t*) ((unsigned int) entry + entry->size + sizeof(entry->size));
	total_memory = (entry->base_addr_high - entry->base_addr_low);
}
```

**Comment by Kfeavel** [comment]
We'll need to take the `type` field into account though as it will inform us where the BIOS memory, APIC, and ISA device space addresses are (if they exist).

**Comment by Kfeavel** [comment]
Partially resolved in #187. Information is available and is printed (also related: #200) but now we need to use that information.

**Comment by Kfeavel** [comment]
Closing issue because the issue at hand is resolved. What we do with the information is not mentioned within the scope provided.
